.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,38 @@
+name: Bug Report
```

Comment:
Gang, I don't know what is correct syntax but could you please add the following drop down:
- product: `dotnet CLI (dotnet new)`, `Visual Studio`, `both`.